lib/provisioner/worker/plugins/automators/docker_automator/docker_automator.rb

```diff
@@ -75,7 +75,8 @@ def search_image(image_name)
     when 0
       fail "Image #{image_name} not found!"
     else
-      fail "More than one image found for #{image_name}!"
+      log.debug "More than one image found for #{image_name}!"
```

Comment:
should this be a warning?  when there is more than one search result, i assume it installs the first?  and the first one could theoretically change over time?